eth2/operation_pool/src/lib.rs

```diff
@@ -96,6 +96,8 @@ impl<T: EthSpec> OperationPool<T> {
     }
 
     /// Get a list of attestations for inclusion in a block.
+    ///
+    /// NOTE: Assumes that all attestations in the operation_pool are valid.
```

Comment:
Rather than "are valid" perhaps "have valid signatures" -- arguably we still check their validity when running `verify_attestation_for_block_inclusion`.

Reply:
Indeed, my bad for bad wording!